templates/docs/patterns/grid/index.md

```diff
@@ -41,167 +41,47 @@ Vanilla has a responsive grid with the following columns and gutters:
 
 <br>
 
-- The page structure must be laid out using rows (`.row`)
-- All content must be within columns (`.col-*`)
+- The page structure must be laid out using rows (`.grid-row`)
+- All content must be within columns (`.grid-col-*`)
 - Only columns should be direct children of a row
 
-Layouts can be created combining rows with different number of columns to an ideal maximum of 4 columns per row. Each column containing text should span a minimum of 3 columns.
+Layouts can be created combining rows with different number of columns to an ideal maximum of 4 columns per row. Each column containing text should span a minimum of 2 columns.
```

Comment:
While this is a new recommended "default" we should clearly mention the legacy grid somewhere on the page so that people looking for docs for existing `row` and `col`s will know where to find it. This should also make it easier to communicate that 2 grids exist and how they differ.

So I'd suggest adding some explanation on top that clearly explains that this page is about NEW 8 column grid introduced in version 4.18 (I guess that's gonna be the next one). 
Followed by some section (or notification component?) mentioning that if you are looking for previous documentation about 12 col grid, you should go to the other page.

And vice-versa - legacy page should clearly mention on top that this is a legacy 12 column grid, and current recommended one is documented on another page.

Reply:
Adjusted - added sections to the top of both pages